src/enums/effectiveDateTypes.ts

```diff
@@ -0,0 +1,4 @@
+export enum EffectiveDateTypes {
+  FUTUREEFFECTIVE = 'futureEffective',
```

Comment:
maybe add an underscore for readability?  i.e. `FUTURE_EFFECTIVE`.

Reply:
Good idea. Will do!

Reply:
looks good!